### PR TITLE
[DOCFIX] remove a space

### PR DESCRIPTION
**Pull request by thu-david**
### What changes are proposed in this pull request?

Please outline the changes and how this PR fixes the issue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

**Comment by thu-david** [comment]
remove a space for pr

File: docs/cn/ufs/Azure-Data-Lake-Gen2.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ priority: 2
 ---
 
 * Table of Contents
-{:toc}
+  {:toc}
```
Comment (apc999) [inline docs/cn/ufs/Azure-Data-Lake-Gen2.md:10]: why space here?

File: docs/cn/ufs/Azure-Data-Lake-Gen2.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ priority: 2
 电脑上应已安装好 Alluxio 程序。如果没有安装，可[编译Alluxio源代码]({{ '/cn/contributor/Building-Alluxio-From-Source.html' | relativize_url }}),
 或直接 [下载已编译好的Alluxio程序]({{ '/cn/deploy/Running-Alluxio-Locally.html' | relativize_url }}).
 
-在将 Azure 数据湖存储与 Alluxio 一起运行前，请在 [Azure 帐户中创建一个新的 Data Lake Storage](https://learn.microsoft.com/zh-cn/azure/storage/blobs/create-data-lake-storage-account) 或使用现有的 Data Lake Storage。这里还应指定需使用的 directory（目录），创建一个新的目录或使用现有目录均可。此外，还需要一个[共享密钥](https://learn.microsoft.com/zh-cn/rest/api/storageservices/authorize-with-shared-key)。
+在将 Azure 数据湖存储与 Alluxio 一起运行前，请在 [Azure 帐户中创建一个新的 Data Lake Storage](https://learn.microsoft.com/zh-cn/azure/storage/blobs/create-data-lake-storage-account) 或使用现有的 Data Lake Storage。这里还应指定需使用的 directory（目录),创建一个新的目录或使用现有目录均可。此外，还需要一个[共享密钥](https://learn.microsoft.com/zh-cn/rest/api/storageservices/authorize-with-shared-key)。
```
Comment (apc999) [inline docs/cn/ufs/Azure-Data-Lake-Gen2.md:19]: `）` vs `)` ? Full-width with half-width characters

**Review by apc999 — CHANGES_REQUESTED** [review]

**Comment by apc999** [comment]
@thu-david could you please address the comments? thanks

**Comment by thu-david** [comment]
单纯为了提pr

**Pull request closed without merging**